gtsfm/loader/olsson_loader.py

```diff
@@ -76,20 +81,25 @@ def __init__(
 
         # stores camera poses (extrinsics) and intrinsics as 3x4 projection matrices
         # 'P' array will have shape (1,num_imgs), and each element will be a (3,4) matrix
-        data = loadmat(cam_matrices_fpath)
+        data = scipy.io.loadmat(cam_matrices_fpath)
+
+        if len(data["P"][0]) != self._num_imgs:
+            raise RuntimeError("Found more or less images than ground truth provides.")
```

Comment:
I think "Number of images not equal to ground truth (images?)" would be a better (less confusing) message.

Reply:
Good point. Updated now.